@here/harp-mapview/lib/MapView.ts

```diff
@@ -1511,7 +1529,97 @@ export class MapView extends THREE.EventDispatcher {
             );
         }
 
-        this.update();
+        this.updateCamera();
+    }
+
+    /**
+     * Returns the camera focus point position in geo-coordinates. If the camera points to an
+     * invalid position will return `null`.
+     */
+    get geoTarget() {
+        return this.m_geoTarget;
+    }
+
+    set geoTarget(target: GeoCoordinates) {
+        const sphericalCoordinates = MapViewUtils.extractSphericalCoordinatesFromLocation(
+            this,
+            this.m_camera,
+            this.geoTarget
+        );
+
+        this.lookAt(
+            target,
+            this.lookAtDistance,
+            THREE.Math.radToDeg(sphericalCoordinates.tilt),
+            THREE.Math.radToDeg(sphericalCoordinates.azimuth)
+        );
+    }
+
+    /**
+     * Return `pitch` value of the current view in degrees.
+     */
+    get pitch(): number {
+        return THREE.Math.radToDeg(MapViewUtils.extractAttitude(this, this.m_camera).pitch);
+    }
+
+    /**
+     * Set `pitch` value of the current view in degrees.
+     */
+    set pitch(pitchDeg: number) {
+        this.setCameraGeolocationAndZoom(this.geoCenter, this.zoomLevel, this.yaw, pitchDeg);
+    }
+
+    /**
+     * Return `yaw` value of the current view in degrees.
+     */
+    get yaw() {
```

Comment:
why not `azimuth`?

Reply:
Ah, just saw it below. I would not add yaw and pitch getters/setters at all but we should stick to azimuth/tilt(or however we want to name them)